core/src/main/scala/org/apache/spark/h2o/backends/external/RestApiUtils.scala

```diff
@@ -180,55 +181,55 @@ trait H2OContextRestAPIUtils extends H2OContextUtils {
 
   /**
     *
-    * @param endpoint An address of H2O node with exposed REST endpoint
-    * @param suffix REST relative path representing a specific call
-    * @param conf H2O conf object
+    * @param endpoint      An address of H2O node with exposed REST endpoint
+    * @param suffix        REST relative path representing a specific call
+    * @param conf          H2O conf object
     * @param skippedFields The list of field specifications that are skipped during deserialization. The specification
     *                      consists of the class containing the field and the field name.
     * @tparam ResultType A type that the result will be deserialized to
     * @return A deserialized object
     */
-  private def query[ResultType : ClassTag](
-       endpoint: URI,
-       suffix: String,
-       conf: H2OConf,
-       skippedFields: Seq[(Class[_], String)] = Seq.empty): ResultType = {
+  private def query[ResultType: ClassTag](
```

Comment:
I have nothing against the original formatting, I just like to use the feature of InteliJ to auto-format. If we have any automatic tool to format it for us, I'm fine with it